modular_zubbers/code/modules/antagonists/bloodsucker/powers/_powers.dm

```diff
@@ -164,7 +164,7 @@
 	// Bloodsuckers in a Frenzy don't have enough Blood to pay it, so just don't.
 	if(bloodsuckerdatum_power.frenzied)
 		return
-	bloodsuckerdatum_power.bloodsucker_blood_volume -= bloodcost
+	bloodsuckerdatum_power.AddBloodVolume(-bloodcost)
```

Comment:
Changed AddBloodVolume to AdjustBloodVolume so it makes more sense, you gotta update your PR so you get those changes